actionpack/lib/action_controller/metal/rescue.rb

```diff
@@ -1,28 +0,0 @@
-# frozen_string_literal: true
-
-module ActionController #:nodoc:
-  # This module is responsible for providing +rescue_from+ helpers
-  # to controllers and configuring when detailed exceptions must be
-  # shown.
-  module Rescue
-    extend ActiveSupport::Concern
-    include ActiveSupport::Rescuable
```

Comment:
Something needs to include `Rescuable`. It is only working now because `StrongParameters` is including and I don't think it should be its responsibility. We either need to move this to `Metal` and force everyone to have this feature out of the box even if they don't need to (which is a no go in my opinion) or keep `Rescue` arounds so `Metal` users can add it if they want.